tests/__init__.py

```diff
@@ -23,6 +23,10 @@
 PERSONINFO_CONTAINER_TGT_DATA = (
     PERSONINFO_BASIC_EXAMPLE_DIR / OUTPUT / "Container-001.transformed.yaml"
 )
+PERSONINFO_TR_COMPILED_MD = PERSONINFO_BASIC_EXAMPLE_DIR / OUTPUT / "personinfo_compiled.md"
+PERSONINFO_TR_COMPILED_PY = PERSONINFO_BASIC_EXAMPLE_DIR / OUTPUT / "personinfo_compiled.py"
+PERSONINFO_DERIVED = PERSONINFO_BASIC_EXAMPLE_DIR / OUTPUT / "personinfo_derived.yaml"
```

Comment:
Added in some expected results for a few of the CLI tests.

In general I don't like this pattern as it can make tests more fragile, but hey! At least you know quickly if anything has changed.